.github/workflows/test-ios-simulator.yaml

```diff
@@ -58,6 +59,21 @@ jobs:
       - name: Set simulator location
         run: xcrun simctl location booted set 52.17469,21.03193
 
+      - name: Set tests to exclude
+        id: set_excluded_tests
+        run: |
+          TESTS_TO_EXCLUDE="
```

Comment:
i'm sure this is WIP but I'll just say that I'd like to see a slack message only with test name (e.g. "android_app_test") instead the full "--exclude integration_test/android_app_test.dart"

Reply:
I'll parse this somehow :)